sound/soc/sof/intel/hda.c

```diff
@@ -380,7 +380,8 @@ static int hda_init_caps(struct snd_sof_dev *sdev)
 	/* create codec instances */
 	hda_codec_probe_bus(sdev, hda_codec_use_common_hdmi);
 
-	hda_codec_i915_put(sdev);
+	if (!HDA_IDISP_CODEC(bus->codec_mask))
+		hda_codec_i915_put(sdev);
```

Comment:
The commit message is not aligned with the code.

Did you mean:

Change HDA probe behaviour slightly so that i915 power is
turned off only if i915 codecs are not found in the initial probe done by
SOF Intel driver, otherwise keep i915 power on until HDA codec driver probe
runs.

Reply:
@plbossart Missed this comment, but agreed. Maybe this wording could be better. I now sent with "power is not turned off .. if codecs are found".